Source/ios/Storyboard/UIResponder+TyphoonOutletTransfer.m

```diff
@@ -0,0 +1,126 @@
+////////////////////////////////////////////////////////////////////////////////
+//
+//  TYPHOON FRAMEWORK
+//  Copyright 2015, Typhoon Framework Contributors
+//  All Rights Reserved.
+//
+//  NOTICE: The authors permit you to use, modify, and distribute this file
+//  in accordance with the terms of the license agreement accompanying it.
+//
+////////////////////////////////////////////////////////////////////////////////
+
+#import "UIResponder+TyphoonOutletTransfer.h"
+#import "NSLayoutConstraint+TyphoonOutletTransfer.h"
+#import <objc/runtime.h>
+
+@implementation UIResponder (TyphoonOutletTransfer)
+
+- (void)transferFromView:(UIView *)view
```

Comment:
I see some problems in this method:

1. Name of method doesn't tell me that it's transfer constraints instead self
2.  Method doing more than 1 things (SRP) and looks a little bit full up